packages/nimble-components/src/table-column/base/cell-view/template.ts

```diff
@@ -0,0 +1,31 @@
+import { type ViewTemplate, html } from '@microsoft/fast-element';
+import type { TableCell } from '../../../table/components/cell';
+import { TableCellView } from '.';
+
+const validateCellViewTemplate = (cellViewTag: string): void => {
+    let instance;
+    try {
+        instance = document.createElement(cellViewTag);
+    } catch (ex) {
+        // Swallow construction error to report a better one
+    }
+    if (!(instance instanceof TableCellView)) {
+        throw new Error(
+            `Cell view tag name (${cellViewTag}) must evaluate to an element extending TableCellView`
+        );
+    }
```

Comment:
Actually this error validation is new behavior, I'll add tests for the custom cell and group header tag validation